.github/workflows/dash_actions.yml

```diff
@@ -0,0 +1,22 @@
+name: Dashboard
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - "**.md"
+      - "**.rst"
+      - "**.ipynb"
+    branches:
+      - main
+
+jobs:
+  call-build-workflow:
+    if: github.event.pull_request.draft == false
```

Comment:
This job should already run only when `pull_request` is `ready_for_review`, I think. Is this extra condition needed?

Reply:
I'm not sure. I think I looked into this a bit previously. I think because different types of PRs are specified this doesn't always hold, but dont remember exactly. I can imagine that this can be simplified, but at this point I don't feel like testing all the options (starting as draft - then marking ready - then back to draft, etc, etc).
I know that like this it triggers when I want it to, so I think should be good enough for the present.